packages/components/CHANGELOG.md

```diff
@@ -204,6 +204,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 + <p-icon name="arrow-right"></p-icon>
 ```
 
+#### Added
+
+- `Carousel` supports new prop `alignHeader`:
```

Comment:
I'd stick to `param-case` here as well like this.
```
- Prop `align-header` for `Carousel`
```
Also this should be added to the already existing `#### Added` section in L12.

Reply:
Fixed